core-plugins/docs/XMLReader-batchsource.md

```diff
@@ -41,8 +41,9 @@ an existing directory.
 
 **tableName:** Table name to keep track of processed file(s).
 
-**temporaryFolder** Existing hdfs folder path having read and write access to the current User, required for internal
-computation of the plugin. Default value is /tmp.
+**temporaryFolder** Existing hdfs folder path having read and write access to the current User, required to store
```

Comment:
Can we have it in a way such that if it doesn't specify then we default to /tmp.